src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java

```diff
@@ -177,11 +173,17 @@ public SBApiResponse profileUpdate(Map<String, Object> request, String userToken
 						getModifiedPersonalDetails(profileDetailsMap.get(changedObj), requestData);
 					}
 				}
+				log.info("Existing Profile Details : " + new Gson().toJson(existingProfileDetails));
+				String schema = getVerifiedProfileSchema();
```

Comment:
schema is not useful in this method... it is required only inside valideJsonAgainstSchema... 
just call the getVerifiedProfileSchema inside that method and avoid passing the value...